src/main/java/liquibase/ext/mongodb/database/MongoConnection.java

```diff
@@ -50,10 +55,13 @@
 @NoArgsConstructor
 public class MongoConnection extends AbstractNoSqlConnection {
 
+    private final Logger log = Scope.getCurrentScope().getLog(getClass());
```

Comment:
You shouldn't set the logger in the class costructors like this in general. While it sometimes happens to work due to how classes are loaded, there are many times where classes are loaded as singletons very early in startup before the logger is correctly configured and/or the logger is otherwise reconfigured between when the class is constructed and when methods want to log. 

So instead you should be getting the log from the scope at the beginning of each method that wants to use it as a general pattern.